teams/roles-and-responsibilities.md

```diff
@@ -1 +1,65 @@
-# Teams: Roles and Responsibilities
+# Ember Core Team Member Roles, Responsibilities, and Expectations
+
+## Participation
+
+As a bare minimum, core team members are expected to:
+
+- Follow the community guidelines- https://emberjs.com/guidelines 
+- Interact thoughtfully:
+    - with the community in an exemplary way - for better or for worse, the “core team” is often perceived as a unit, not as individuals
+    - with other members of the core teams in a respectful way (do we need to say this?)
+- Lead the official projects through some combination of
+    - Triaging issues
+    - Reviewing PRs and mentoring community contributors
+    - Answering community questions (Discord, [Discuss](https://discuss.emberjs.com/), Stack Overflow, etc)
+    - Supporting community documentation efforts
+    - Writing blog posts for emberjs.com
+    - Championing RFCs
+    - Supporting the release process
+- Attend weekly team meetings
+- Be reliable
+    - If you say you can do something, do it
+    - If you can’t do it (life happens- and we are all volunteers) let the team know
+
+
+## Annual Commitments
+
+The expectations for participation is that every core team member is expected to participate, in the way(s) that they define for themselves. 
+
+Each team member should define the ways in which they will participate, and will revisit this on an annual basis. This will help to ensure that the core teams continue to challenge ourselves and grow. It will also help us identify what type of core team member we are missing, so we can help ourselves grow a balanced team. 
+
+There are many different ways to participate as a core team member:
+
+- Writing new feature code
+- Refactoring old code to improve it
+- Writing documentation
+- Technical Mentorship
+    - Regularly answering questions in chat & on the forum
+    - Training new potential maintainers - working to “grow” the next group of Ember core team members
+- Speaking about Ember at Conferences and Meetups
+- Writing RFCs
+- Leading projects (i.e., writing the quest issue, providing project management, hosting strike team calls)
+
+Note: “thought leadership” is not a form of participation. Every core team member is expected to already be capable of this before joining the core team (whether they choose to voice thought leadership or not).  
+
+Lack of participation (either by not defining goals, or not honoring the commitments made) is one of the indicators that alumnization should be considered. However, if goals have been defined, but a team member has a major life event, a sabbatical may be considered. 
```

Comment:
> a team member has a major life event

O_o isn't life a major life event? What kinds of things are we talking about here?

Reply:
I believe our google-fu to be the same, but here's a common definition for life event: 

"Life events are defined as discrete experiences that disrupt an individual's usual activities, causing a substantial change and readjustment. Examples of life events include marriage, divorce, illness or injury, and changing or losing a job."